src/shared/migrations/1624449896646-add-user-entity-with-relations.ts

```diff
@@ -0,0 +1,77 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { POSTS_SCHEMA } from '../entity.model';
+
+export class addUserEntityWithRelations1624449896646
+  implements MigrationInterface {
+  name = 'addUserEntityWithRelations1624449896646';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "${POSTS_SCHEMA}"."votes" DROP CONSTRAINT "FK_26c647863c296d49e748b5ef98f"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "${POSTS_SCHEMA}"."votes" RENAME COLUMN "user_id" TO "userId"`,
```

Comment:
I thought underscores are considered the common/best practices for DBs. please check the best practices.

Reply:
It is auto generated by typeorm. It needs some workaround to change it. If it is necessity, I will change it.

Reply:
please create a ticket to introduce naming strategy as snake_case
https://www.npmjs.com/package/typeorm-naming-strategies

Reply:
After further investigating typeorm docs, all auto generated column names have been changed to follow snake_case. So, there is no need to add the mentioned library.